src/tests/JIT/Performance/JittedMethodsCountingTest/JittedMethodsCountingTest.cs

```diff
@@ -9,14 +9,30 @@
 
 public class JittedMethodsCountingTest
 {
-    private const int MAX_JITTED_METHODS_ACCEPTED = 50;
+    private const int MAX_JITTED_METHODS_ACCEPTED = 120;
 
     [Fact]
     public static int TestEntryPoint()
     {
+        // If DOTNET_ReadyToRun is disabled, then this test ought to be skipped.
+        if (!IsReadyToRunEnvSet())
+        {
+            Console.WriteLine("\nThis test is only supported in ReadyToRun scenarios."
+                              + " Skipping...\n");
+            return 100;
+        }
+
         string appName = "HelloWorld.dll";
         string jitOutputFile = "jits.txt";
 
+        // DOTNET_JitStdOutFile appends to the file in question if it exists.
+        // This can potentially cause issues, so we have to make sure it is
+        // created exclusively for this test's purpose.
+        if (File.Exists(jitOutputFile))
+        {
+            File.Delete(jitOutputFile);
+        }
+
         // For adding any new apps for this test, make sure their success return
         // code is 0, so we can universally handle when they fail.
         int appResult = RunHelloWorldApp(appName, jitOutputFile);
```

Comment:
Should we include a check for the ```appResult``` value in the expression for the exit code?

Reply:
I don't think that would be necessary. We're already marking the test as failed and returning 101 if the app's exit code is different from 0 (i.e. something went wrong there).